### ISSUE TITLE
timestamp input and output formats

### ISSUE THREAD

**Issue by djw1149** [dnsdb/dnsdbq#19]
The -A and -B timestamp input arguments expect YYYY-MM-DD[HH:MM:SS] or the relative form: %dw%dd%dh%dm%ds.  Of the three output forms, json outputs timestamps in Unix epoch format.  I suggest adding an option to have json output human readable timestamp format too.

csv:
```
"2017-02-14 23:15:53","2017-03-01 23:16:09",,,11,,"106.119.53.92.in-addr.arpa.","PTR","bbn.com."
```

json:
```
{"count":11,"time_first":1487114153,"time_last":1488410169,"rrname":"106.119.53.92.in-addr.arpa.","rrtype":"PTR","rdata":"bbn.com."}
```

dns:
```
;; record times: 2017-02-14 23:15:53 .. 2017-03-01 23:16:09
;; count: 11
106.119.53.92.in-addr.arpa.  PTR  bbn.com.
```

**Comment by vixie** [comment]
djw1149 wrote:
> The -A and -B timestamp input arguments expect YYYY-MM-DD[HH:MM:SS] or
> the relative form: %dw%dd%dh%dm%ds. Of the three output forms, json
> outputs timestamps in Unix epoch format. I suggest adding an option to
> have json output human readable timestamp format too.

what option letter do you suggest? in any case, send a pull request.

-- 
P Vixie

**Comment by djw1149** [comment]
Attempted to fix in MR #21 but merge was rejected.  Abandoning this issue.
